### PR TITLE
DDPB 819 add hint text to NDR

### PR DESCRIPTION
**Pull request by lauratennant**
Hint texts added where needed in Actions, Assets, Debts,
Income/Benefits, Visits/Care

### CODE REVIEW

File: src/AppBundle/Resources/translations/odr-debts.en.yml [hunk 1]
```diff
@@ -19,6 +19,7 @@ editPage:
   htmlTitle: "New deputy report - debts | GOV.UK"
   pageSectionTitle: Debts
   pageSectionDescription: Please tell us more about %client%'s debts.
+  hint: If %client% has a mortgage on a property, use the Assets section to tell us about it
```
Comment (Biggsen) [inline src/AppBundle/Resources/translations/odr-debts.en.yml]: @lauratennant You can only add the 'hint:' below 'label:', not anywhere else. This won't get picked up by the backend.

File: src/AppBundle/Resources/translations/odr-assets.en.yml [hunk 2]
```diff
@@ -106,6 +106,7 @@ form:
       label: What is %client%’s share of the property?
     isSubjectToEquityRelease:
       label: Is this property subject to an equity release scheme?
+      hint: This means that %client%'s stake, or equity, in the property has been used to generate a lump sum or income
```
Comment (elvisciotti) [inline src/AppBundle/Resources/translations/odr-assets.en.yml]: anytime there is a %client% variable that wasn't there, it might be not rendered

Comment (Biggsen) [inline src/AppBundle/Resources/translations/odr-assets.en.yml]: I've gone through all the changes and fixed missing hint text and all the %client% which wasn't showing.

I'm merging this now.

**Comment by Biggsen** [comment]
All looking very good.

File: src/AppBundle/Resources/translations/odr-assets.en.yml (hunk 2 above)
Comment (elvisciotti) [inline src/AppBundle/Resources/translations/odr-assets.en.yml]: ok

**Pull request merged**